core/interfaces/UnitPropertiesInterface.ts

```diff
@@ -10,5 +11,6 @@ export interface UnitPropertiesInterface {
   armor: number,
   hitPoints: number,
   playerId: string,
+  position: PositionEnum,
```

Comment:
Rename position and positionEnum to something better describing its use. Maybe BoardStatusEnum and the other to BattleStatusEnum